opencensus/stats/view_data.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.stats import aggregation
+from opencensus.stats import aggregation_data
+from opencensus.stats import measure
+from opencensus.stats import view
+from opencensus.tags import tag_value
+from opencensus.tags import tag_map
+from datetime import datetime
+
+
+class ViewData(object):
+    """View Data is the aggregated data for a particular view
+
+    :type view:
+    :param view: The view associated with this view data
+
+    :type start_time: datetime
+    :param start_time: the start time for this view data
+
+    :type end_time: datetime
+    :param end_time: the end time for this view data
+
+    :type rows: list(:class: '~opencensus.tags.tag.Tag', :class: '~opencensus.stats.aggregation_data.AggregationData')
+    :param rows: the collected value for a specific set of tags
+
+    """
+    def __init__(self, view, start_time, end_time, rows=None):
+        self._view = view
+        self._start_time = start_time
+        self._end_time = end_time
+        self._rows = rows if rows is not None else {}
+        self._tag_value_aggregation_map = {}
+        self._tag_map = {}
+
+    @property
+    def view(self):
+        """the current view in the view data"""
+        return self._view
+
+    @property
+    def start_time(self):
+        """the current start time in the view data"""
+        return self._start_time
+
+    @property
+    def end_time(self):
+        """the current end time in the view data"""
+        return self._end_time
+
+    @property
+    def rows(self):
+        """the current rows in the view data"""
+        return self._rows
+
+    @property
+    def tag_value_aggregation_map(self):
```

Comment:
The key of this map should be different combinations of `TagValue`s, rather than single `TagValue`.